examples/raspberrypi/rp2xxx/src/dma.zig

```diff
@@ -36,24 +36,26 @@ pub fn main() !void {
 
     const channel = dma.claim_unused_channel().?;
 
-    channel.trigger_transfer(
-        @intFromPtr(dst[0..dst.len].ptr),
-        @intFromPtr(hello[0..hello.len].ptr),
-        hello.len,
-        .{
-            .enable = true,
-            .read_increment = true,
-            .write_increment = true,
-            .data_size = .size_8,
-            .dreq = .permanent,
-        },
-    );
+    try channel.setup_transfer(dst[0..dst.len], hello[0..hello.len], .{
+        .trigger = true,
+        .enable = true,
+    });
+    channel.wait_for_finish_blocking();
+
+    if (!std.mem.eql(u8, &dst, hello))
+        @panic("dma failed");
 
+    var value: u8 = 0x41;
+    try channel.setup_transfer(&dst, &value, .{
+        .trigger = true,
+        .enable = true,
+    });
     channel.wait_for_finish_blocking();
 
-    var i: u32 = 0;
-    while (true) : (i += 1) {
-        std.log.info("{s}", .{dst});
+    if (!std.mem.allEqual(u8, &dst, 0x41))
+        @panic("dma failed");
+
+    while (true) {
```

Comment:
Maybe we should actually do something in the loop instead of basically just spinning

Reply:
but there is nothing to do, those examples from what i understand those examples are only to show the API usage, and test the code